lightcurve/alcdef.py

```diff
@@ -0,0 +1,84 @@
+from django.contrib import admin
+from django.db import models
+from django.http import HttpResponse
+
+class AlcdefWriter(models.Model):
+    def __init__(self):
+        self.metadata = {
+            'REVISEDDATA': '',
+            'OBJECTNUMBER': '',
+            'OBJECTNAME': '',
+            'MPCDESIG': '',
+            'CONTACTNAME': '',
+            'CONTACTINFO': '',
+            'OBSERVERS': '',
+            'OBSLONGITUDE': '',
+            'OBSLATITUDE': '',
+            'SESSIONDATE': '',
+            'SESSIONTIME': '',
+            'FILTER': '',
+            'MAGBAND': '',
+            'STANDARD': '',
+            'DIFFERMAGS': '',
+            'LTCTYPE': '',
+            'LTCDAYS': '',
+            'LTCAPP': '',
+            'REDUCEDMAGS': '',
+            'UCORMAG': '',
+            'OBJECTRA': '',
+            'OBJECTDEC': '',
+            'PHASE': '',
+            'PABL': '',
+            'PABB': '',
+            'CICORRECTION': '',
+            'CIBAND': '',
+            'CITARGET': '',
+            'PUBLICATION': '',
+            'BIBCODE': '',
+            'DELIMITER': '',
+            'COMMENT': '',
+        }
+        self.starid = 1 # ID for metadata of comparison star
+        self.comments = []
+        self.datas = []
+
+    def set(self, key, value):
+        self.metadata[key] = value
+
+    def add_comment(self, comment):
+        del self.metadata['COMMENT']
```

Comment:
How come we have a default COMMENT metadata as a string?  Is the idea that the user can explicitly set a comment string if they want?

Reply:
The alcdef examples I viewed have all the allowed keywords, even if its empty. So I made sure I have COMMENT in the self.metadata in case it's empty. If there are comments, then add_comment() remove the empty self.metadata['COMMENT']  and creates a string later on that contain the comments. I have commented out line 49 for now. Please read notes below.